src/ray/common/task/task_util.h

```diff
@@ -87,7 +88,11 @@ class TaskSpecBuilder {
   TaskSpecBuilder() : message_(std::make_shared<rpc::TaskSpec>()) {}
 
   /// Build the `TaskSpecification` object.
-  TaskSpecification Build() { return TaskSpecification(message_); }
+  TaskSpecification Build() {
+    auto task_spec = TaskSpecification(message_);
+    RAY_LOG(DEBUG) << "Build task " << task_spec.DebugString();
```

Comment:
I prefer adding a new log in `CoreWorker::CreateActor` and whatever other places that are needed. Because there will be more useful information like "Submitting normal task ..." and "Submitting actor. task ..."